packages/components/src/Inline/Inline.stories.tsx

```diff
@@ -39,7 +40,7 @@ const meta = {
         type: { summary: 'text' },
         defaultValue: { summary: 'undefined' },
       },
-      options: ['top', 'center', 'bottom'],
+      options: [...Object.keys(alignment.vertical.alignmentY), 'input'].flat(),
```

Comment:
here is also `evenly`, `around` etc applied but this has no effect. There is no `align-items:evenly`